### ISSUE TITLE
Summary command failing with reflect: call of reflect.Value.NumField on zero Value error

### ISSUE THREAD

**Issue by myprivatealaska** [pb33f/openapi-changes#86]
Summary command is failing when rendering the comparison between the following two specs: 
`original.yml` 

```
openapi: "3.0.1"
paths:
  /api/v2/sourcing/views/{viewId}:
    get:
      operationId: getViewById
      parameters:
        - name: viewId
          in: path
          description: View ID
          required: true
          schema:
            type: integer
            format: int64
      responses:
        "200":
          description: Successful Request
          content:
            application/json:
              schema:
                $ref: "#/components/schemas/view"
components:
  schemas:
    view:
      type: object
      description: View model
      properties:
        name:
          type: string
          description: The name of the view
          example: "Example"
```

`modified.yml`

```
openapi: 3.0.1
paths:
  "/api/v2/sourcing/views/{viewId}":
    get:
      operationId: getViewById
      parameters:
        - description: View ID
          in: path
          name: viewId
          required: true
          schema:
            format: int64
            type: integer
      responses:
        '200':
          content:
            application/json:
              schema:
                "$ref": "#/components/schemas/view"
          description: Successful Request
components:
  schemas:
    view:
      description: View model
      example:
        name: Example
      properties:
        name:
          description: The name of the view
          example: Example
          type: string
      type: object
```
`openapi-changes summary original.yml modified.yml` throws the following error:

```
panic: reflect: call of reflect.Value.NumField on zero Value

goroutine 1 [running]:
reflect.flag.mustBe(...)
        /usr/local/go/src/reflect/value.go:233
reflect.Value.NumField({0x0?, 0x0?, 0x45270f?})
        /usr/local/go/src/reflect/value.go:2062 +0x85
github.com/pb33f/openapi-changes/builder.exploreTreeObject(0xc0003aa380, {0xbe41a0?, 0x0?})
        /opt/openapi-changes/builder/tree.go:42 +0xc5
github.com/pb33f/openapi-changes/builder.BuildTreeMapNode(0xc0003aa2a0, {0xbba320?, 0xc000464068?, 0xba0329?})
        /opt/openapi-changes/builder/tree.go:350 +0xa5
github.com/pb33f/openapi-changes/builder.exploreTreeObject(0xc0003aa2a0, {0xbe41a0?, 0xc000464000?})
        /opt/openapi-changes/builder/tree.go:236 +0x2010
github.com/pb33f/openapi-changes/builder.BuildTreeMapNode(0xc0003aa230, {0xbba320?, 0xc0003a8868?, 0xb81ae4?})
        /opt/openapi-changes/builder/tree.go:350 +0xa5
github.com/pb33f/openapi-changes/builder.exploreTreeObject(0xc0003aa230, {0xbe4da0?, 0xc0003a8860?})
        /opt/openapi-changes/builder/tree.go:236 +0x2010
github.com/pb33f/openapi-changes/builder.DigIntoTreeNode[...](0xc0003aa1c0, {0xbe4da0, 0xc0004183f0?, 0x10e8ca8?}, {0xc0003ad080?, 0xa}, 0x1, 0x0)
        /opt/openapi-changes/builder/tree.go:300 +0x2ea
github.com/pb33f/openapi-changes/builder.exploreTreeObject(0xc0003aa1c0, {0xbe51a0?, 0xc000418380?})
        /opt/openapi-changes/builder/tree.go:141 +0x3185
github.com/pb33f/openapi-changes/builder.BuildTree({0xbe51a0, 0xc000418380})
        /opt/openapi-changes/builder/tree.go:35 +0x9d
github.com/pb33f/openapi-changes/cmd.printSummaryDetails({0xc00039a620, 0x2, 0xc0001eda28?})
        /opt/openapi-changes/cmd/summary.go:404 +0x2bd
github.com/pb33f/openapi-changes/cmd.runLeftRightSummary({0x7ffe0f313f47, 0x1a}, {0x7ffe0f313f62, 0x14}, 0x4?, 0x1007208?, {0x0, 0x0}, 0x0?)
        /opt/openapi-changes/cmd/summary.go:326 +0x8c5
github.com/pb33f/openapi-changes/cmd.GetSummaryCommand.func1(0xc0001ac100?, {0xc00007eae0, 0x2, 0xd27790?})
        /opt/openapi-changes/cmd/summary.go:228 +0x975
github.com/spf13/cobra.(*Command).execute(0xc000004600, {0xc00007ea80, 0x2, 0x2})
        /go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:983 +0xabc
github.com/spf13/cobra.(*Command).ExecuteC(0x1746560)
        /go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1115 +0x3ff
github.com/spf13/cobra.(*Command).Execute(...)
        /go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1039
github.com/pb33f/openapi-changes/cmd.Execute({0xd284fd?, 0x4012b0?}, {0xd284fd?, 0xd35bce?}, {0xc0000587a0?, 0xc0000061a0?})
        /opt/openapi-changes/cmd/root.go:51 +0xb0
main.main()
        /opt/openapi-changes/openapi-changes.go:29 +0x158
```

**Comment by daveshanley** [comment]
will fix this.

**Comment by daveshanley** [comment]
resolved in `v0.0.53`
